extension.neon

```diff
@@ -26,13 +26,7 @@ parameters:
         - Illuminate\Http\Request
     excludes_analyse:
         - *.blade.php
-    ignoreErrors:
-        - '#Result of function abort \(void\) is used#'
-        - '#Call to an undefined method Illuminate\\Support\\HigherOrder#'
-        - '#Method App\\Exceptions\\Handler::render\(\) should return Illuminate\\Http\\Response but returns Symfony\\Component\\HttpFoundation\\Response#'
```

Comment:
That one can be fixed in existing projects and is no longer required in new projects, Laravel cleaned up their act.